app/bin/tools/backfill_packageversions.dart

```diff
@@ -7,25 +7,36 @@ import 'dart:async';
 import 'package:appengine/appengine.dart';
 import 'package:args/args.dart';
 import 'package:gcloud/db.dart';
+import 'package:pool/pool.dart';
 
 import 'package:pub_dartlang_org/frontend/models.dart';
 import 'package:pub_dartlang_org/frontend/service_utils.dart';
 
 final _argParser = new ArgParser()
+  ..addOption('concurrency',
+      abbr: 'c', defaultsTo: '1', help: 'Number of concurrent processing.')
   ..addOption('package', abbr: 'p', help: 'The package to backfill.');
 
 Future main(List<String> args) async {
```

Comment:
Unrelated, by maybe give it a description... 3 lines saying what this does... backfill what from where...